File: VizsgaGyak/cars_20240505/CarRental-desktop/Car.cs
namespace CarRental_desktop
{
    internal class Car
    {
        public Car(int id, string licensePlateNumber, string brand, string model, int dailyCost)
        {
            Id = id;
            LicensePlateNumber = licensePlateNumber;
            Brand = brand;
            Model = model;
            DailyCost = dailyCost;
        }

        public Car()
        {
            Id = 0;
            LicensePlateNumber = "";
            Brand = "";
            Model = "";
            DailyCost = 0;
        }

        public int Id { get; }

        public string LicensePlateNumber { get; }

        public string Brand { get; }

        public string Model { get; }

        public int DailyCost { get; }

        public override string ToString()
        {
            return $"{LicensePlateNumber} - {Brand} - {Model} - {DailyCost} Ft";
        }
    }
}

File: VizsgaGyak/cars_20240505/carrentalbackend/app/Models/Car.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Car extends Model
{
    use HasFactory;

    public $fillable = ['license_plate_number','brand','model','daily_cost'];

    protected $hidden = ['created_at','updated_at'];
}
